src/scala/com/github/johnynek/bazel_deps/DepsModel.scala

```diff
@@ -1523,8 +1523,37 @@ object ResolverType {
   case object Aether extends ResolverType("aether") {
     override def optionsDoc: Option[Doc] = None
   }
-  case object Coursier extends ResolverType("coursier") {
-    override def optionsDoc: Option[Doc] = None
+  case class Coursier(
+    hashInHttpHeaders: Option[Boolean]
+  ) extends ResolverType("coursier") {
+    def getHashInHttpHeaders: Boolean = hashInHttpHeaders.getOrElse(false)
+
+    override def optionsDoc: Option[Doc] = {
+
+      val items = List(
+        (
+          "hashInHttpHeaders",
+          hashInHttpHeaders.map(b => Doc.text(s"$b"))
+        ),
+      ).sortBy(_._1)
```

Comment:
I guess this was copied, but we don't need to sort a list of length 1. but you can leave it if you add a comment that this is here so we don't forget if we add more options.